packages/xod-client-electron/src/debugger/sendToSerialMiddleware.js

```diff
@@ -20,7 +20,8 @@ export default ({ getState }) => next => action => {
 
   const isTweakActionType =
     action.type === client.NODE_UPDATE_PROPERTY ||
-    action.type === client.TWEAK_PULSE_SENT;
+    action.type === client.TWEAK_PULSE_SENT ||
+    action.type === client.NODE_PROPERTY_TWEAKED;
```

Comment:
I think that `NODE_PROPERTY_TWEAKED` name will cause some confusion in the future, but I can't think of a better alternative yet. Probably, it should reflect that it is an intermediate value and is not intended to end up in a redux store.

Reply:
What about `SEND_TWEAK_VALUE_TO_INTERACTIVE_SESSION`?

Reply:
`SEND_TWEAK_VALUE_TO_INTERACTIVE_SESSION` — no, please. Action names are the _facts_. System parts may _subscribe_ to _facts_ they are interested in. `SEND_TWEAK_VALUE_TO_INTERACTIVE_SESSION` is not much better than calling `sendValueToDebugger` right inside the component.

I see what @evgenykochetkov means but it requires a minor clarification. Use present continuous, for example (`NODE_PROPERTY_TWEAKING`)… Or to match the existing name: `NODE_UPDATING_PROPERTY`. Better it would be `NODE_PROPERTY_UPDATING` but we already have a somewhat malformed name for `NODE_UPDATE_PROPERTY`.

Reply:
@nkrkv @evgenykochetkov just renamed these two action types.
- `NODE_UPDATE_PROPERTY` -> `NODE_PROPERTY_UPDATED` (see last commit)
- `NODE_PROPERTY_TWEAKED` -> `NODE_PROPERTY_UPDATING` (fixup)